src/dog.sol

```diff
@@ -134,9 +137,9 @@ contract Dog /* is LibNote */ {
             (,rate, spot,, dust) = vat.ilks(ilk);
             require(spot > 0 && mul(ink, spot) < mul(art, rate), "Dog/not-unsafe");
 
-            uint256 room = sub(hole, dirt);
+            uint256 room = min(sub(Hole, Dirt), sub(milk.hole, milk.dirt));
 
-            // Test whether the remaining space in the hole is dusty
+            // Test whether the remaining space in the Hole is dusty
```

Comment:
This comment really isn't accurate since it's either the remainder of `Hole` or the per-ilk `hole`. IMO the intent of the code is clear and the comment could actually be removed, but if we keep it, it should be made fully correct.

Reply:
Changed a bit the comments. Please let me know if fine now or feel free to tweak them.